applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.h

```diff
@@ -104,6 +104,45 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) EmbeddedSkinVisualizationProcess :
     ///@name Life Cycle
     ///@{
 
+    /**
+     * @brief Get the Default Settings object
+     * Static method to get the default settings inside the contructor
+     * @return Parameters The parameters object containing the default settings
+     */
+    static Parameters GetDefaultSettings();
```

Comment:
This is in the baseclass now, I think a different name, please check

Reply:
In the base class there is `GetDefaultParameters`. In any case it cannot be `override` as I need it to be `static`.

Reply:
Having said this, maybe it's better to keep a different name.